x86.c

```diff
@@ -40,6 +40,10 @@ static const unsigned char co_swap_function[4096] = {
 };
 
 #ifdef _WIN32
+  #if defined(__GNUC__) || defined(_MSC_VER)
```

Comment:
Is there some non-GCC-like, non-MSVC-like C compiler we're worried about here?

Reply:
For Windows? Not likely. Worst case, it'll just not include the Valgrind header.

Though I must question why that file includes valgrind.h on MSVC when [amd64.c does not](https://github.com/higan-emu/libco/pull/39/files/8731d1d7dc5a5d296ef901edeabc3d10d97b9c66#diff-eac206c4a92b9980c9a90bd2fcf11e108f80c6d826becd178aeaf0f2dded376aR69).

Reply:
...it's because msvc x86 supports inline asm, but msvc amd64 does not, right?

May want to add a comment, though.

Reply:
I just used the same logic as the ifdefs in valgrind.h

Reply:
I've added a code comment about this.

It's possible (at least theoretically) to run Valgrind under Wine on a Windows exe file.